testsuites/syncgateway/functional/tests/test_rollback.py

```diff
@@ -117,30 +113,20 @@ def test_rollback_server_reset(params_from_base_test_setup, sg_conf_name):
     # Delete vbucket and restart server
     cb_server.delete_vbucket(66, "data-bucket")
     cb_server.restart()
-
-    max_retries = 20
+    max_retries = 50
     count = 0
     while count != max_retries:
         # Try to get changes, sync gateway should be able to recover and return changes
-        try:
-            # A changes since=0 should now be in a rolled back state due to the data loss from the removed vbucket
-            # Seth should only see the docs not present in vbucket 66, unlike all the docs as above.
-            changes = client.get_changes(url=sg_url, db=sg_db, since=0, auth=seth_session)
+        # A changes since=0 should now be in a rolled back state due to the data loss from the removed vbucket
+        # Seth should only see the docs not present in vbucket 66, unlike all the docs as above.
+        changes = client.get_changes(url=sg_url, db=sg_db, since=0, auth=seth_session)
+        changes_ids = [change["id"] for change in changes["results"] if not change["id"].startswith("_user")]
+        log_info("length of Changes ids are {} and vbuckets {}".format(len(changes_ids), num_vbuckets))
+        if len(changes_ids) == (num_vbuckets - 1):
```

Comment:
Doing the additional verification that the missing doc is actually the one from the rolled back vbucket would be a good idea here (to ensure there's not something else going on).

Reply:
yeah , that is good point . Will add the verification for that